trie.h

```diff
@@ -15,7 +15,7 @@
 #include <utility>
 #include <cstddef>
 #include <string>
-#include <hat-trie/include/tsl/htrie_map.h>
+#include <htrie_map.h>
```

Comment:
This needs to be conditionally included, since support for hattrie is optional.

Reply:
I think conditional build without hattrie is not ready ;)

Reply:
Too bad.